lua/lvim/lsp/peek.lua

```diff
@@ -33,7 +33,7 @@ local function create_floating_file(location, opts)
     false
   )
   local width, height = vim.lsp.util._make_floating_popup_size(contents, opts)
-  opts = vim.lsp.util.make_floating_popup_options(width, height, opts)
+  opts = vim.lsp.util.make_floating_popup_options(math.max(width, 30), math.max(height, 10), opts)
```

Comment:
maybe it's safer to use `vim.F.if_nil` instead? otherwise you need to validate you're not passing `nil`

```suggestion
  local if_nil = vim.F.if_nil
  opts = vim.lsp.util.make_floating_popup_options(if_nil(width, 30), if_nil(height, 10), opts)
```